packages/tooltip/src/Tooltip/Tooltip.spec.tsx

```diff
@@ -694,6 +694,20 @@ describe('packages/tooltip', () => {
       });
     });
 
+    test('onMouseDown events should fire', async () => {
```

Comment:
added this spec to preserve the existing testing pattern for mouse events. I think it would be ideal to introduce a puppeteer test to check the browser navigation behavior but don't think it makes sense to introduce with this ticket

Reply:
you can create a ticket for this!

Reply:
also, can you test that if `onClick` is passed along with `onMouseDown` it's also called?

Reply:
ignore my comment above!